array-methods/foreach/src/__tests__/1_forEach_exercises.test.js

```diff
@@ -132,7 +146,13 @@ describe("Exercise 3", () => {
 `;
 
 const logUserInformation = (array, callBack) => {
-  // Your solution here
+  array.forEach(user => {
+    Object.keys(user).forEach(property => {
```

Comment:
```suggestion
    Object.entries(user).forEach([ key, value ] => {
```
Nice chaining! This was perfectly effective. If you know you need both `key` an `val` you can leverage `.entries` and destructuring over `.keys` to save yourself a few keystrokes (but I almost always find myself using keys instead).